include/hdr/hdr_histogram.h

```diff
@@ -64,6 +85,25 @@ int hdr_init(
     int significant_figures,
     struct hdr_histogram** result);
 
+/**
+ * Initialise the hdr_histogram expecting preallocated memory, for pg_stat_statements.
+ *
+ * @param lowest_discernible_value The smallest possible value that is distinguishable from 0.
+ * Must be a positive integer that is >= 1. May be internally rounded down to nearest power of 2.
+ * @param highest_trackable_value The largest possible value to be put into the
+ * histogram.
```

Comment:
nit: good to define what will happen if the recorded value is greater than this value, discarded or what.